app/views/servers/_form.html.erb

```diff
@@ -19,6 +19,10 @@
     <%= f.label :url %><br>
     <%= f.text_field :url, class: "form-control" %>
   </div>
+  <div class="field form-group">
+    <%= f.label :backend %><br>
```

Comment:
The whole page is about defining a backend server. I think in this context, it makes more sense to call this "Server type" rather than "Backend".